src/resources/Api.ts

```
@@ -277,23 +277,25 @@ export class Api {
-            Name: domain.name,
-            Type: 'A',
-            AliasTarget: {
-              HostedZoneId: {
-                'Fn::GetAtt': [domainNameLogicalId, 'HostedZoneId'],
-              },
-              DNSName: {
-                'Fn::GetAtt': [domainNameLogicalId, 'AppSyncDomainName'],
-              },
-              EvaluateTargetHealth: false,
-            },
+            RecordSets: ['A', 'AAAA'].map(Type => ({
```

ESLint

[error] Replace `Type` with `(Type)`

(prettier/prettier)